examples/postgres/README.md

```diff
@@ -2,11 +2,23 @@
 
 This example shows how to access a PostgreSQL database from Spin component.
 
+## Prerequisite: Postgres
+
+This example assumes postgres is running and accessible locally via its standard 5432 port.
+
+We suggest running the `postgres` Docker container which has the necessary postgres user permissions
+already configured. For example:
+
+```
+docker run --rm -h 127.0.0.1 -p 5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust postgres
+```
+
 ## Spin up
 
-From example root:
+Then, run the following from the root of this example:
 
 ```
+export PGUSER=postgres PGHOST=localhost
 createdb spin_dev
 psql -d spin_dev -f db/testdata.sql
 RUST_LOG=spin=trace spin build --up
```

Comment:
Consider removing the RUST_LOG while we're in here?  It seems like it's only likely to clutter the output?  (I realise it's a separate change of course.  Not a must-do if you want to keep the PR focused.)

Reply:
Done.